### PR TITLE
LPS-76855

### PR DESCRIPTION
**Pull request by pedroqueiroz94**

### CODE REVIEW

**Comment by pedroqueiroz94** [comment]
ci:test

File: modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-date/src/main/resources/META-INF/resources/date.soy [hunk 1]
```diff
@@ -34,7 +34,7 @@
 	<div class="{$formGroupClass}" data-fieldname="{$name}">
 		{if $showLabel}
 			{let $labelAttributes kind="attributes"}
-				class="control-label
+				class="
 					{if $readOnly}
```
Comment (clovisdasilvaneto) [inline modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-date/src/main/resources/META-INF/resources/date.soy]: Hey @alinedoleron , Can you rename the $readOnly variable to $disabled variable?

File: modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-numeric/src/main/resources/META-INF/resources/numeric.soy [hunk 2]
```diff
@@ -28,15 +28,15 @@
 
 	<div class="form-group{$visible ? '' : ' hide'} liferay-ddm-form-field-numeric {$tip ? 'liferay-ddm-form-field-has-tip' : ''}" data-fieldname="{$name}">
 		{if $showLabel}
-			<label class="control-label{$readOnly ? ' disabled' : ''}" for="{$name}">
+			<label class="{$readOnly ? ' disabled' : ''}" for="{$name}">
```
Comment (clovisdasilvaneto) [inline modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-numeric/src/main/resources/META-INF/resources/numeric.soy]: Hey @alinedoleron , Can you rename the $readOnly variable to $disabled variable?

File: modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-select/src/main/resources/META-INF/resources/select.soy [hunk 3]
```diff
@@ -139,15 +139,15 @@
 	{@param? required: bool}
 	{@param? tip: string}
 
-	<label class="control-label{$readOnly ? ' disabled' : ''}" for="{$name}">
+	<label class="{$readOnly ? ' disabled' : ''}" for="{$name}">
```
Comment (clovisdasilvaneto) [inline modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-select/src/main/resources/META-INF/resources/select.soy]: Hey @alinedoleron , Can you rename the $readOnly variable to $disabled variable?

File: modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-date/src/main/resources/META-INF/resources/date.soy (hunk 1 above)
Comment (alinedoleron) [reply modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-date/src/main/resources/META-INF/resources/date.soy]: Hi @clovisdasilvaneto, I believe this name comes from back end.

File: modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-numeric/src/main/resources/META-INF/resources/numeric.soy (hunk 2 above)
Comment (alinedoleron) [reply modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-numeric/src/main/resources/META-INF/resources/numeric.soy]: Hi @clovisdasilvaneto, I believe this name comes from back end.

File: modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-select/src/main/resources/META-INF/resources/select.soy (hunk 3 above)
Comment (alinedoleron) [reply modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-select/src/main/resources/META-INF/resources/select.soy]: Hi @clovisdasilvaneto, I believe this name comes from back end.

**Comment by interaminense** [comment]
LGTM.

**Comment by pedroqueiroz94** [comment]
ci:retest

**Comment by pedroqueiroz94** [comment]
@rodrigo-e-c-souza, can you take a look at the tests here? Thanks!

**Comment by rodrigo-e-c-souza** [comment]
ci:test

**Comment by rodrigo-e-c-souza** [comment]
@pedroqueiroz94 failures here are related to known issues

**Comment by pedroqueiroz94** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by pedroqueiroz94** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/54896. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**